map/models.py

```diff
@@ -1,10 +1,12 @@
 from django.db import models
 from djgeojson.fields import PointField
 
+from search.models import CraigslistLocation
+
 
 class CityStreetSpot(models.Model):
 
-    c_id = models.CharField(max_length=200)
+    c_id = models.ForeignKey(CraigslistLocation, on_delete=models.CASCADE)
```

Comment:
Is it possible to use `OneToOneRelationship` instead of directly dealing with the key?

https://docs.djangoproject.com/en/2.2/topics/db/examples/one_to_one/

I'm not sure it's one to one but there are also `ManyToMany` and `OneToMany`, .etc.